src/supabase/queries.ts

```diff
@@ -0,0 +1,45 @@
+import { supabase } from './client';
+
+// Function to add a single tree
+export async function addTree(species: string) {
+  const { error } = await supabase.rpc('add_tree', { species });
+
+  if (error) {
+    throw new Error(`Error adding tree: ${error.message}`);
+  }
+}
+
+// Function to add multiple trees
+export async function addMultipleTrees(
+  trees: { species: string; quantity: number }[],
+) {
+  const { error } = await supabase.rpc('add_multiple_trees', {
```

Comment:
You can pass in the two arguments separately instead of using JSON.stringify here.